### PR TITLE
[DOC-3784] Delegate 23.09.80804 release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
Delegate 23.09.80804 release notes

For reviewers, previews are available:

[Delegate Release Notes (NextGen)](https://651575f934fa691e5c8beb9e--harness-developer.netlify.app/release-notes/delegate)

[Delegate Release Notes (FirstGen)](https://651575f934fa691e5c8beb9e--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/fg-delegate)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 1]
```diff
@@ -20,51 +20,65 @@ To identify the cluster that hosts your account, open Harness FirstGen, go to **
...
+This release does not include early access features.
 
 ### Fixed issues
 
-- Fixed an issue when using multiple HTTP Helm Chart repositories that could lead to an increase in CPU utilization on the delegate due to background connector validation tasks. This was caused by running Helm repository update during the validation tasks. (CDS-76433, ZD-48363)
-
-- With an earlier update, delegates tried to create a Kubernetes runner, which created an API client using the Kubernetes config. Shell delegates tried to fetch the local config. GKE configurations with expired credentials resulted in an error. (PL-40631, ZD-48998, ZD-49702)
-
-   This issue is fixed. Harness catches the exception and continues with delegate startup.
+- You can no longer update `Delegatelp` or `DelegateName` using the delegate update API. (PL-40795, ZD-44419)
```
Comment (pratmit) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:45]: Is it DelegateIP ? I am reading as DelegateLP.

**Review by pratmit — COMMENTED** [review]
Commented

File: docs/first-gen/firstgen-release-notes/fg-delegate.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md]: @pratmit Nice catch. Updated with a commit.

**Review by pratmit — DISMISSED** [review]

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 2]
```diff
@@ -20,51 +20,65 @@ To identify the cluster that hosts your account, open Harness FirstGen, go to **
...
+
+- The Spring Boot library is upgraded to version 2.7.14. (PL-40810)
+
+##### Early access features
+
+This release does not include early access features.
+
+##### Fixed issues
+
+- Fixed an issue when using multiple HTTP Helm Chart repositories that could lead to an increase in CPU utilization on the delegate due to background connector validation tasks. This was caused by running Helm repository update during the validation tasks. (CDS-76433, ZD-48363)
```
Comment (SudheendraKatte) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:72]:
```suggestion
- Fixed an issue when using multiple HTTP Helm Chart repositories that could lead to an increase in CPU utilization on the delegate due to background connector validation tasks. This was caused by running the Helm repository update during the validation tasks. (CDS-76433, ZD-48363)
```

File: release-notes/delegate.md [hunk 3]
```diff
@@ -28,12 +28,92 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- You can now reference secret values in JSON files by using XPATH. Support is available for AWS Secret Manager, Azure Key Vault, GCP Secret Manager, and HashiCorp Vault. (PL-41063)
+
+### Early access features
+
+This release does not include any new early access features.
+
+### Fixed issues
+
+- The Kustomize 3.5.4 binary is now removed from the immutable delegate and all Kustomize tasks are routed via Kubectl binary. (CDS-58893, ZD-48553)
```
Comment (SudheendraKatte) [inline release-notes/delegate.md:52]:
```suggestion
- The Kustomize 3.5.4 binary is now removed from the immutable delegate, and all Kustomize tasks are routed via the Kubectl binary. (CDS-58893, ZD-48553)
```

File: release-notes/delegate.md [hunk 4]
```diff
@@ -28,12 +28,92 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- In certain scenarios for ECS Blue Green deployments, the Green application was not rolling back. We have added functionality to handle this scenario. We now consistently roll back the Green service in ECS Blue Green deployments. (CDS-76795, ZD-49005, ZD-49919)
+
+- Fixed an issue where ShellScript WinRM deployments would not honor the configured timeout. For example, the step would time out by default in 30 minutes even when the configured timeout was 1 day. Now the WinRM session timeout will be set to 30 minutes or the timeout configured for the step (if more than 30 minutes). (CDS-78219, ZD-48180, ZD-49871)
+
+- Fixed an issue with Artifactory artifact fetches in the pipeline, when the artifact path was in a nested directory and also a regex. (CDS-78278, ZD-50030)
+
+- Resolved an issue when copying config files from BitBucket repositories if a folder path was specified instead of a file path. (CDS-78344, ZD-49489)
+
+- The output of the Kubernetes Dry Run step did not generate a valid Kubernetes manifest due to masking of the secrets values (CDS-78507).
```
Comment (SudheendraKatte) [inline release-notes/delegate.md:62]:
```suggestion
- The output of the Kubernetes Dry Run step did not generate a valid Kubernetes manifest due to the masking of the secrets values (CDS-78507).
```

**Review by SudheendraKatte — APPROVED** [review]
Approved with minor suggestions

File: docs/first-gen/firstgen-release-notes/fg-delegate.md (hunk 2 above)
Comment (brian-f-harness) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:72]:
```suggestion
- Fixed an issue when using multiple HTTP Helm Chart repositories that could lead to an increase in CPU utilization on the delegate due to background connector validation tasks. This was caused by running the Helm repository update during the validation tasks. (CDS-76433, ZD-48363)
```

**Pull request merged**